main.py

```diff
@@ -120,7 +120,7 @@ def BoardInit(args):
         ### Logic Modules ###
         has_muse_ppg = master_board_id in (BoardIds.MUSE_2_BOARD, BoardIds.MUSE_S_BOARD)
         
-        fft_size=2048
+        fft_size= 64 * 10

```

Comment:
Let's add a TODO comment to this so we know why this is hard-coded for the moment